src/main/java/com/redhat/devtools/intellij/tektoncd/ui/hub/HubModel.java

```diff
@@ -83,8 +85,10 @@ public HubModel(Project project, Tkn tkn, String namespace, List<String> tasks,
     public List<HubItem> getAllHubItems() {
         if (allHubItems.isEmpty()) {
             try {
-                allHubItems = retrieveAllHubItems().get();
-            } catch (InterruptedException | ExecutionException e) {
+                List<Language> languages = new LanguageRecognizerBuilder().build().analyze(project.getBasePath());
```

Comment:
`Project.getBasePath()` is not working in all cases. Project in IntelliJ is a workspace that contains a collection of Modules (projects). So I suspect you can an empty project in directory A and add a Maven or Gradle module from directory B.
But the current implementation will work for 80% of the user cases, I think.